src/app/[locale]/(main)/rest-client/page.tsx

```diff
@@ -1,22 +1,61 @@
 'use client';
 
 import { redirect } from 'next/navigation';
+import { lazy, Suspense, useEffect } from 'react';
 import { useAuthState } from 'react-firebase-hooks/auth';
+import { useDispatch } from 'react-redux';
 
 import Loader from '@/components/Loader';
-import RestClient from '@/components/restClient/RestClient';
+import { decodeFromBase64, queryParamsToHeaders } from '@/components/restClient/utils/urlUtils';
 import { auth } from '@/lib/auth/firebase.config';
+import { setHeader, updateRestClientFormField } from '@/lib/store/slice/restClientFormSlice';
+
+const RestClient = lazy(() => import('@/components/restClient/RestClient'));
```

Comment:
Можно оставить конструкцию lazy() + suspense 

[инфа здесь ](https://nextjs.org/docs/app/guides/lazy-loading)

или использовать 
```suggestion
const RestClient = dynamic(() => import('@/components/restClient/RestClient'), {
  loading: () => <Loader />,
});
```

Reply:
<img width="874" height="596" alt="image" src="https://github.com/user-attachments/assets/08788742-88e1-49ad-a085-80c9b33569f2" />
+ надо оставить че-та одно

Reply:
заменила на dynamic везде

Reply:
<img width="448" height="500" alt="image" src="https://github.com/user-attachments/assets/750c77aa-dc8f-4aaa-b75d-a1f71cc78b00" />